src/main/groovy/pl/allegro/tech/build/axion/release/domain/VersionWithPosition.groovy

```diff
@@ -24,4 +26,8 @@ class VersionWithPosition {
     boolean forcedVersion() {
         return position == null
     }
+
+    void setSnapshotVersion(boolean isSnapshotVersion) {
```

Comment:
change siganture -> `void asSnapshotVersion()` which sets `isSnapshotVersion` to true, i don't think true/false argument is needed.